reflex/reflex.py

```diff
@@ -440,7 +440,11 @@ def deploy(
         config.app_name,
         "--app-name",
         help="The name of the App to deploy under.",
-        hidden=True,
+    ),
+    app_id: str = typer.Option(
+        None,
+        "--app-id",
+        help="The ID of the App to deploy under.",
```

Comment:
```suggestion
        help="The ID of the App to deploy over.",
```

I think the terminology should indicate that the given app id will be overwritten if you specify it like this